### PR TITLE
Move deepcell-label desktop into its own project

### PR DESCRIPTION
**Pull request by willgraf**
## What
* Move `/desktop` project into its own repository.
* Move `/browser` code into a `deepcell_label` flask app folder.
* Move `create_app` into `deepcell_label/__init__.py`.
* Updated webpack configs, `pytest.ini`, `README.md`, and the GitHub Actions workflows to use the new module name.
* Resolved several import issues using the new module name.
* Fixes typo in Watershed command instructions.

## Why
* Promote browser-based DeepCell Label as the primary entry point for users looking to use the tool. 
* Resolve confusion between the two different projects sharing the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,68 @@
-# DeepCell Label: Data Curation Tools for DeepCell.
-[![Actions Status](https://github.com/vanvalenlab/deepcell-label/workflows/browser/badge.svg)](https://github.com/vanvalenlab/deepcell-label/actions)
+# Browser-Based Application of DeepCell Label for Data Curation
 
-DeepCell Label is a segmentation and tracking tool used for human-in-the-loop data curation. It displays lineage data along with raw and annotated images. The output files prepare this information as training data for DeepCell.
+This is an version of DeepCell Label that runs on a browser. The browser-based DeepCell Label application can be run locally or deployed to AWS Elastic Beanstalk.
+
+Many key and mouse operations are the same between desktop and browser versions, but these versions are not guaranteed to share the same features. See the "Controls" section for an up-to-date list of features.
```
Comment (tddough98) [inline README.md:5]: I don't think we should mention the desktop version here. Instead, "See the "Controls" section for an up-to-date list of features." should be enough

File: .ebignore [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+
+# JavaScript Dependencies
+node_modules/
```
Comment (willgraf) [inline .ebignore:118]:
```suggestion
node_modules/

```

**Review by tddough98 — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -112,4 +121,9 @@ venv.bak/
 # mypy
 .mypy_cache/
 
-node_modules
+# VS Code
+.vscode
+
+# JavaScript Dependencies
+node_modules/
+package-lock.json
```
Comment (willgraf) [inline .gitignore:129]:
```suggestion
package-lock.json
yarn.lock
```

File: README.md [hunk 4]
```diff
@@ -1,25 +1,68 @@
...
-python3 caliban.py [input file location]
+pip install -r requirements.txt
 ```
 
-**Accepted file types:**
-DeepCell Label can open .trk files or .npz files. .npz files must contain two zipped files corresponding to raw images and annotated images. If the files are not named 'raw' and 'annotated' or 'X' and 'y', the first file in the .npz will be opened as the raw images. Raw and annotated images must both be in 4D arrays in the shape (frames, y, x, channels or features).
+## Run browser application in development mode
+```bash
+python3 application.py
+```
```
[the remark this answers is not in the document]
Comment (willgraf) [reply README.md:21]: I think development is fine for default and we can update it during deployment.

File: README.md [hunk 5]
```diff
@@ -1,25 +1,68 @@
...
+## To use docker-compose for local development
+Add your AWS credentials to ```docker-compose.yaml```.
```
[the remark this answers is not in the document]
Comment (willgraf) [reply README.md]: No, i think that is overly complicated. Instead, we can update the docker-compose to read from the environment. I've just pushed that change.

File: README.md [hunk 6]
```diff
@@ -1,25 +1,68 @@
...
+To interact with the local mysql database:
+```
+sudo docker exec -it browser_db_1 bash
+mysql -p
+```
+When finished:
+```
+sudo docker-compose down
+```
+(optional)
```
[the remark this answers is not in the document]
Comment (willgraf) [reply README.md:54]: `docker-compose down` takes down the all the running containers.

The optional step, `docker system prune --volumes` is an extra cleanup step to remove any stray data that was mounted for persistence.

File: README.md [hunk 7]
```diff
@@ -28,140 +71,72 @@ Files can be edited using keyboard operations.
 
 DeepCell Label's default setting allows operations to be carried out quickly and easily on existing segmentations. The actions that can modify cell labels and/or lineage information are:
```
[the remark this answers is not in the document]
Comment (willgraf) [reply README.md]: Yeah let's update it after the fact.

**Review by tddough98 — APPROVED** [review]
Looks stunning, a great improvement in project organization

File: .ebextensions/options.config [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+  aws:elasticbeanstalk:application:environment:
+    FLASK_APP: application
+    FLASK_ENV: production
+
+  aws:elasticbeanstalk:container:python:
+    WSGIPath: application.py
+
+  aws:elasticbeanstalk:container:python:staticfiles:
+    /static/: deepcell_label/static/
```
[the remark this answers is not in the document]
Comment (willgraf) [reply .ebextensions/options.config:11]: I got thrown for a loop as well because `deepcell_label/static` failed.

**Pull request merged**